settings.gradle.kts

```diff
@@ -69,6 +69,7 @@ include(
 
 include(
     ":feature:widget:message-list",
+    ":feature:widget:shortcut",
```

Comment:
I think we should avoid using the generic name "shortcut".

We only have one type of shortcut now. But could have multiple ones in the future, e.g. shortcuts to open a compose dialog with a particular sender identity and a given recipient address.

So we could make this `:feature:widget:account-shortcut` or `:feature:shortcut:account`.

Reply:
While there might be different shortcut types eventually. They all need to share the same maximum limit of shortcuts allowed by the system. I don't see a reason to further split this up into individual shortcut modules, until we have a concrete use case.